internal/config/config.go

```diff
@@ -8,17 +8,18 @@ import (
 )
 
 type Config struct {
-	MetricsLabels              []string       `mapstructure:"cis-metrics-labels"`
-	ScanInterval               time.Duration  `mapstructure:"scan-interval"`
-	ScanJobNamespace           string         `mapstructure:"scan-job-namespace"`
-	ScanJobServiceAccount      string         `mapstructure:"scan-job-service-account"`
-	ScanNamespaces             []string       `mapstructure:"namespaces"`
-	ScanNamespaceExcludeRegexp *regexp.Regexp `mapstructure:"scan-namespace-exclude-regexp"`
-	ScanNamespaceIncludeRegexp *regexp.Regexp `mapstructure:"scan-namespace-include-regexp"`
-	ScanWorkloadResources      []string       `mapstructure:"scan-workload-resources"`
-	TrivyImage                 string         `mapstructure:"trivy-image"`
-	TrivyCommand               TrivyCommand   `mapstructure:"trivy-command"`
-	ActiveScanJobLimit         int            `mapstructure:"active-scan-job-limit"`
+	MetricsLabels                  []string       `mapstructure:"cis-metrics-labels"`
+	ScanInterval                   time.Duration  `mapstructure:"scan-interval"`
+	ScanJobNamespace               string         `mapstructure:"scan-job-namespace"`
+	ScanJobServiceAccount          string         `mapstructure:"scan-job-service-account"`
+	ScanJobTTLSecondsAfterFinished int            `mapstructure:"scan-job-ttl-seconds-after-finished"`
```

Comment:
Would `int32` here improve/avoid the `int32` casting later?

```suggestion
	ScanJobTTLSecondsAfterFinished int32            `mapstructure:"scan-job-ttl-seconds-after-finished"`
```

Reply:
Go flags doesn't support int32. 😒 It might work, and we can try.